mdn/templates/mdn/feature_page_list.jinja2

```diff
@@ -98,10 +143,10 @@
 
 {% endblock content %}
 
-{% block block_js_extra %}
+{% block body_js_extra %}
 <script>
 $(function () {
-  $('#issue-header').tooltip()
+  $('[data-toggle="tooltip"]').tooltip()
```

Comment:
Something something about abusing `data-*` attributes with semantics that belong in the `class` list.